### DISCUSSION TITLE
General Question on older SDL game

### DISCUSSION THREAD

**Discussion by lonnietc** [MethanePowered/MethaneKit#112] [Q&A] [answered]
Hello,

Hope that you are doing well today.

There is a bit dated SDL2 game (Tesseract 3D Engine, from Cube2) that I am interested in trying to update to a modern rendering engine so that I can also learn in the process about this area as well, but it seems to use SDL2 and OpenGL for the rendering part.

It seemed to make sense to me to consider a modern rendering engine that could also do Vulkan but from a good framework that covered more.

This will initially be run on Windows, but can also have builds for Linux and MacOS as well.

Do you think that Methane Kit is capable and good for this purpose?

Thanks and have a great day

**Comment by egorodet** [comment, marked as answer]
Hello @lonnietc,

Methane Kit is currently in Alpha stage of development and has only a basic rendering functionality and its API is pretty unstable (frequently changing). Some important functionality, like Compute pipeline or "Execute Indirect" support is still missing, but planned for development. It should be fine to use Methane Kit for some graphic experiments or code contributions, but it is not ready for full fledged game development yet.
